### PR TITLE
docs: update Oracle Cloud Infrastructure platform setup documentation

### PR DESCRIPTION
**Pull request by Djelibeybi**
Signed-off-by: Avi Miller <avi.miller@oracle.com>

Updated the Oracle Cloud Infrastructure (OCI) platform setup documentation to be more useful with links to the appropriate OCI docs for more details.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by rvennam — APPROVED** [review]

**Comment by Djelibeybi** [comment]
I'm not sure why the cla/google bot didn't add the `cla: yes` label to this PR and I can't seem to find a command to retrigger that check, so if someone knows the right place to gently poke the bot, that would be appreciated. :L)

**Comment by ericvn** [comment]
@Djelibeybi There is an underlying change in the CLA bot to make things better. I think part of that process is missing. I'm not sure if the label is required or just the All contributors are covered under a CLA with Google` which you have. I've raised the problem and hopefully this will be fixed soon.

**Comment by ericvn** [comment]
This PR also helps with  https://github.com/istio/istio.io/issues/6513 as it at least defines OKE at the top of the document.

File: content/en/docs/setup/platform-setup/oci/index.md [hunk 1]
```diff
@@ -11,35 +11,88 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
-This page was last updated January 4, 2019.
+This page was last updated September 20, 2021.
 
 {{< boilerplate untested-document >}}
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/oci/index.md:16]: Was this document manually tested to verify it works? I suspect it was. Then we may want to remove this template as well.

Comment (Djelibeybi) [reply content/en/docs/setup/platform-setup/oci/index.md:16]: It was tested by Oracle, yes.

**Pull request merged**